toolkit/templates/layouts/_base_page.html

```diff
@@ -20,6 +20,12 @@
   {% include "toolkit/proposition-header.html" %}
 {% endblock %}
 
+{% block after_header %}
+  {% if not current_user.user_research_opted_in and not 'seen_user_research_message' in request.cookies %}
```

Comment:
Where there's a string i.e. `seen_user_research_message` that needs to be the same in two files, and we can't define a constant somewhere, then we should probably have a comment at each end referencing the other thing? (i.e. between here and the JS file)